tcms/core/tests/test_admin.py

```diff
@@ -123,14 +123,19 @@ def setUpTestData(cls):
         cls.regular_user.set_password("password")
         cls.regular_user.save()
 
+        cls.regular_user2 = UserFactory()
+        cls.regular_user2.is_staff = True
+        cls.regular_user2.set_password("password")
+        cls.regular_user2.save()
```

Comment:
The 3 lines above are not really needed, the test just needs the record in the DB. Doesn't care about the rest.